### ISSUE TITLE
[LE] When nodes starts request the current template database from all members

### ISSUE THREAD

**Issue by dangershony** [stratisproject/FederatedSidechains#102]

**Comment by dangershony** [comment]
It seems a member can reconstruct the template db just by looking at the blockchain.
So this task may actually be to be able to build the tx template.

This will consists of 

- blockhash
- current leader
- multisg trx
- signed inputs

All this can be brebuild just by looking at the blockchain.

**Comment by monsieurleberre** [comment]
Agreed, I will look in a bit more depth at how exactly we can do this without broadcast, but it feels very possible :)

**Comment by bokobza** [comment]
@dangershony what does [LE] stand for?

I don't see the point of this. the nodes all know how to build a template, why would they need to look at the blockchain for that?
